dev-client/src/screens/CreateSiteScreen/components/CreateSiteView.tsx

```diff
@@ -66,17 +64,14 @@ export const CreateSiteView = ({
 
   const onSave = useCallback(
     async ({...form}: FormState) => {
-      const {coords, ...site} = validationSchema.cast(form);
-      const createdSite = await createSiteCallback({
-        ...site,
-        ...parseCoords(coords),
-      });
+      const {...site} = validationSchema.cast(form);
+      const createdSite = await createSiteCallback({...site, elevation});
```

Comment:
You said you plan for a future PR to handle the case where the user has changed the lat/long in the form, right?

Reply:
Yes.